docs: add self hosted 1.10 release notes (#435)
okteto/docs: src/content/self-hosted/install/releases.mdx

id: releases
---

## 1.10.0
5 July 2023

> 

Reviewer: I think we have to include a `removal notice` for `1.23` like we did [here](https://www.okteto.com/docs/self-hosted/install/releases/#removal-notice) for `1.21`.

We didn't do it for `1.22` but I think it is good to explicitly call it out. WDYT @omnomagonz ?

### Features

> - Now you can select an expiration period when creating a new Personal Access Token. /* 6690 */

Reviewer: ```suggestion
- Now you can select an expiration period when creating a new Personal Access Token.
```

Should we remove all these comments?

Reviewer: I would remove them. They don't provide values for customers reading the release notes and customers cannot access to them (not like in the open source product). Internally, we have ways of getting that information again without complexity

Reviewer: Oh, ok, my comment was about making it visible in the release notes.

I like that idea 👏

- Upgraded ingress-nginx to [4.6](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.7.0). /* 6625 */

### Improvements

- Kubernetes Annotations can now be added to: /* 6689, 6668 */
  - Default self-signed wildcard certificate via `.Values.wildcardCertificate.annotations`.
  - Internal self-signed webhook certificate via `.Values.webhook.internalCertificate.annotations`.
  - Dockerconfig via `.Values.secret.annotations`.
- Kubernetes Labels can now be added to: /* 6668 */
  - Dockerconfig via `.Values.secret.labels`.

### Bugfixes

- Fixed an installation step warning that was displayed incorrectly when using the default `.Values.subdomain` (`localtest.me`). /* 6717 */
- Cert-manager annotations are now allowed in developer ingresses. /* 6697 */
- When a license is not specified, the Starter plan is displayed instead of Unknown. /* 6671 */
- Upgraded the Okteto CLI version to [2.16.5](https://github.com/okteto/okteto/releases/tag/2.16.5). /* 6762, 6764 */
  - When using `okteto deploy --remote ...` with private certificates, internal registry requests didn't use them correctly. /* okteto/3743, okteto/3758 */

> - Fixed some rendering errors within Okteto External Resources markdown documents. /* 6639 */

Reviewer: What are those `some` errors? Could we be more specific? I think being specific is better for users to understand what specifically was fixed to see if it affects them somehow

id: releases
---

## 1.10.0
5 July 2023

## Removal Notice

Kubernetes 1.23 is no longer supported.

### Features

> - Now you can select an expiration period when creating a new Personal Access Token. <!-- 6690 -->

Reviewer 2: ```suggestion
- Now you can select an expiration period when creating a new [Personal Access Token](docs/cloud/personal-access-tokens/#creating-a-personal-access-token). <!-- 6690 -->
```

Non-blocking suggestion to add more utility to the entry 🙂

- Administrators have the ability to wake up, put to sleep, or mark any preview environment as keep-awake. <!-- 6662, 6622 -->
- Upgraded ingress-nginx to [4.6](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.7.0). <!-- 6625 -->

### Improvements

- Kubernetes Annotations can now be added to: <!-- 6689, 6668 -->
  - Default self-signed wildcard certificate via `.Values.wildcardCertificate.annotations`.
  - Internal self-signed webhook certificate via `.Values.webhook.internalCertificate.annotations`.
  - Dockerconfig via `.Values.secret.annotations`.
- Kubernetes Labels can now be added to: <!-- 6668 -->
  - Dockerconfig via `.Values.secret.labels`.

### Bugfixes

> - Fixed an installation step warning that was displayed incorrectly when using the default `.Values.subdomain` (`localtest.me`). <!-- 6717 -->

Reviewer 2: What was the specific warning? It could be a "nice to have" to add a bit more specificity here, but non-blocking

- Cert-manager annotations are now allowed in developer ingresses. <!-- 6697 -->
- When a license is not specified, the Starter plan is displayed instead of Unknown. <!-- 6671 -->
- Upgraded the Okteto CLI version to [2.16.5](https://github.com/okteto/okteto/releases/tag/2.16.5). <!-- 6762, 6764 -->

>   - When using `okteto deploy --remote ...` with private certificates, internal registry requests didn't use them correctly. <!-- okteto/3743, okteto/3758 -->

Reviewer 2: ```suggestion
  - When using `okteto deploy --remote ...` with private certificates, internal registry requests will now work correctly. <!-- okteto/3743, okteto/3758 -->
```

Non-blocking suggestion to focus more on the positive outcome vs the negative impact from before

> - When an invalid repo URL is used in the Launch Dev Environment view, pressing the enter key won't close the modal. <!-- 6609 -->

Reviewer 2: ```suggestion
- We've added additional form validation to the Launch Dev Environment modal so invalid repo URLs will no longer be accepted and allow the modal to close. <!-- 6609 -->
```

Non-blocking suggestion that I think is a bit more direct and explicit
